dom0/securedrop-handle-upgrade

```diff
@@ -47,6 +47,23 @@ if [[ $TASK == "prepare" ]]; then
 		fi
 	fi
 
+    # For the SDW Whonix VMs, shut them down, so we can upate the TemplateVM settings.
+    # We shut down sd-proxy first, since its netvm is sd-whonix, which won't
+    # shutdown if a client is connected.
+	if qvm-check --quiet sd-proxy; then
+		BASE_TEMPLATE=$(qvm-prefs sd-proxy template)
+		if [[ ! $BASE_TEMPLATE =~ "buster" ]]; then
+			qvm-shutdown --wait sd-proxy
+		fi
+	fi
+
+	if qvm-check --quiet sd-whonix; then
+		BASE_TEMPLATE=$(qvm-prefs sd-whonix template)
+		if [[ ! $BASE_TEMPLATE =~ "buster" ]]; then
```

Comment:
Since TemplateVM for sd-whonix should be whonix-gw-{14,15}, I think he we would want to compare `BASE_TEMPLATE` to '14'. otherwise sd-whonix will shut down on every run.